skills_taxonomy_v2/pipeline/skills_extraction/skills_naming_utils.py

```diff
@@ -100,18 +167,22 @@ def clean_cluster_descriptions(sentences_data):
             replace_ngrams(sentence, ngram_words) for sentence in cluster_docs_cleaned
         ]
 
-        cluster_descriptions[cluster_num] = cluster_docs_clean
+        cluster_descriptions[cluster_number] = cluster_docs_clean
```

Comment:
I think this is overwriting every iteration of the for loop? as `cluster_number` is just one number